extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/streams/StreamResource.java

```diff
@@ -97,7 +98,8 @@ public void sseJson2(Sse sse, SseEventSink sink) throws IOException {
     @GET
     @RestStreamElementType(MediaType.APPLICATION_JSON)
     public Multi<Message> multiJson() {
-        return Multi.createFrom().items(new Message("hello"), new Message("stef"));
+        return RestMulti.from(Multi.createFrom().items(new Message("hello"), new Message("stef")))
```

Comment:
I'm wondering if it should be a supplier. With a supplier, the multi will only be called during the subscription. Sure, it won't be much different, and RestEasy Reactive will subscribe just after the call, but it can make a difference.

Reply:
@cescoffier but we are not subscribing here are we? We are just just using a "container" to pass the `Multi` around, or am I missing something?

Reply:
So, the question is when does the underlying multi be created? Here of during the subscription (that is called deferred multi)

Reply:
It's created at the same time a user would create a Multi if they just wanted to return it for normal streaming. So no difference with the existing functionality